src/main/resources/application.properties

```diff
@@ -1,10 +1,15 @@
-url.inventory=
-url.auth=
+#inventory service details
+url.inventory.host=https://t12345.beta2.kibocommerce.com
+url.inventory.path=/kibo.inventory.webapi/commerce/inventory
 
+#auth service details
+url.auth.host=https://t12345.beta2.kibocommerce.com
```

Comment:
can we have just one host and multiple paths?

Reply:
done